src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlServiceImpl.java

```diff
@@ -60,7 +60,7 @@ public boolean canAccessCaseTypeWithCriteria(final CaseTypeDefinition caseType,
 
         boolean hasAccess = hasAccessControlList(accessProfiles, criteria, caseType.getAccessControlLists());
         if (!hasAccess) {
-            LOG.info(NO_ROLE_FOUND, "caseType",
+            LOG.debug(NO_ROLE_FOUND, "caseType",
```

Comment:
In last 7 days this is logged ~8.5m times, it's part of the standard filtering so we shouldn't need to log everything.